GraphWidget.py

```
@@ -0,0 +1,60 @@
+import random
+import threading
```

Ruff (0.8.2)

`threading` imported but unused

Remove unused import: `threading`

(F401)